src/database/queries/userGroupChangemakerPermissions/deleteOne.sql

```diff
@@ -0,0 +1,7 @@
+UPDATE user_group_changemaker_permissions
+SET not_after = now()
+WHERE
+	keycloak_organization_id = :keycloakOrganizationId
+	AND permission = :permission::permission_t
```

Comment:
I realize this is copy pasta but why do we cast to permission_t here (and elsewhere) and not everywhere.

I'd say lets not cast unless we have to.